crates/sui-indexer-alt-consistent-store/src/db/mod.rs

```diff
@@ -213,6 +213,26 @@ impl Db {
         })
     }
 
+    /// Get the names of all column families in the database.
+    pub(crate) fn column_family_names(&self) -> Vec<String> {
+        // Return the known column families from the schema plus the watermark CF
+        vec![
+            "balances".to_string(),
+            "object_by_owner".to_string(),
+            "object_by_type".to_string(),
+            WATERMARK_CF.to_string(),
+        ]
+    }
+
+    /// Access the underlying RocksDB database for metrics collection.
+    pub(crate) fn db<F, R>(&self, f: F) -> R
```

Comment:
I'd like to avoid this too, as we're leaking an implementation detail of the `Db` here, but that may be more difficult -- let's see.